### DISCUSSION TITLE
'include_cached` error even after following documentation

### DISCUSSION THREAD

**Discussion by riyadhrazzaq** [mmistakes/minimal-mistakes#2651] [Q&A] [answered]
I installed the theme by following [Gem based method](https://mmistakes.github.io/minimal-mistakes/docs/quick-start-guide/#gem-based-method). Everything runs perfectly on local machine. 

But error message from github on page build:
```
The page build failed for the `master` branch with the following error:

The tag `include_cached` on line 15 in `/_layouts/default.html` is not a recognized Liquid tag. For more information, see https://docs.github.com/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#unknown-tag-error.
```
Gemfile:
```
gem "minimal-mistakes-jekyll"
gem "jekyll-include-cache"
```
`_config.yml` snippet, the complete file can be accessed at this [link](https://github.com/riyadhrazzaq/blog/blob/master/_config.yml)
```
url: "https://riyadhrazzaq.github.io"
baseurl: "/blog"
repository: "riyadhrazzaq/blog"

plugins:
  - jekyll-include-cache
  - jekyll-paginate
  - jekyll-sitemap
```
I haven't change any `_includes` or `_layouts` file. Thanks in advance.

**Comment by mmistakes** [comment, marked as answer]
Gem version of the theme doesn’t work with GitHub Pages.
Follow the remote theme method instead. You also don’t need the includes, layouts, and Sass files in your repo.

https://mmistakes.github.io/minimal-mistakes/docs/quick-start-guide/#remote-theme-method

Forking the theme's repo is not advised at all. Instead fork the starter repo and then configure/add your content.
https://github.com/mmistakes/mm-github-pages-starter/generate

**Comment by riyadhrazzaq** [comment]
used the starter repo, everything works now. thank you.
